### DISCUSSION TITLE
Error exporting dataset/project

### DISCUSSION THREAD

**Discussion by ksharova** [asreview/asreview#1253] [Q&A] [answered]
I am trying to export datasets (tried in all available formats) or a project and this keeps popping up: 
"send_file() got an unexpected keyword argument 'download_name' Please try again."   

I am using a CSV file downloaded from Scopus and it is now fully labeled by ASReview. 

What can be done about that and how to proceed?

**Comment by J535D165** [comment, marked as answer]
https://github.com/asreview/asreview/issues/1166#issuecomment-1160296552

Another app might have downgraded your flask dependency.

**Reply by ksharova** [reply]
Thanks for your response! What should I do in this case?

**Reply by Rensvandeschoot** [reply]
see https://github.com/asreview/asreview/issues/1166#issuecomment-1160296552

**Reply by ksharova** [reply]
Thank you very much for the link, sorry I haven't noticed it in your pevious response! 
It worked with this code `pip install --force-reinstall flask==2.0`, thanks again!
